src/app/pages/register/register.component.scss

```diff
@@ -0,0 +1,133 @@
+body {
+    background-image: url(/assets/background_pictures/register.jpg);
+    background-size: cover;
+}
+
+.inscription-form {
```

Comment:
Verify all namen of classes not in full english verify pls (registration best than inscription fr)